### ISSUE TITLE
Is there any method to hide Index in interface files? 

### ISSUE THREAD

**Issue by arvinxx** [tgreyuk/typedoc-plugin-markdown#177]
A Case is here: [Case](https://arvinxx.github.io/umi-plugin-extensions/#/umi-plugin-extensions/api/interfaces/extensionsplugin-config)

In this case, I generate doc into markdown and the document framework dumi will generate document pages by markdown.

![image](https://user-images.githubusercontent.com/28616219/103485960-4010ad00-4e35-11eb-8383-d063c8e9cb76.png)

But by default typedoc will generate index.It will work well in most case. But in my case, it will be a noise because the dumi will show the toc direct by the header level.

So my question is there any way to hide this index in interface files?

Related Issue: https://github.com/TypeStrong/typedoc/issues/1448

**Comment by tgreyuk** [comment]
typedoc-plugin-markdown@3.3.0

There is an option `navigationEnabled` that assumes the document has a navigation/sidebar won't print indexes for interfaces.

**Comment by arvinxx** [comment]
> typedoc-plugin-markdown@3.3.0
> 
> There is an option `navigationEnabled` that assumes the document has a navigation/sidebar won't print indexes for interfaces.

Thanks! This is really useful.
But a suggestion is the option name might be `enableIndex` or `enableTOC` ? And by default `True` to generate Index/TOC.

I was a litte confused when I see this option and used the `navigationEnabled: false` at first but didn't work.And then changed it to`navigationEnabled: true`.

**Comment by tgreyuk** [comment]
@arvinxx thanks - yes i agree its confusing will update as per your suggestion.

**Comment by tgreyuk** [comment]
@arvinxx updated with a slightly different syntax as needed to cover all use cases. Some of the TOC contain links to other pages. This will hide the in-page/anchor TOC as relevant to interfaces, classes enums etc. Hope this makes more sense.

Available in 3.4.0

```
--hideInPageTOC<boolean>
Do not render in-page table of contents items. Defaults to false.
```
